rednotebook/util/markup.py

```diff
@@ -273,6 +291,10 @@ def _get_config(target, options):
 
         config['postproc'].append([COLOR_ESCAPED, r'\\textcolor{\2}{\1}'])
 
+        # Links to entry references are not supported in TeX export - we rewrite them here.
+        config['preproc'].append([r'\[(?P<name>.+)\s+(?P<date>\d{4}-\d{2}-\d{2})\]',
+                                  r'\g<name> (\g<date>)'])
```

Comment:
This repeats the code for `if is_exporting_to_file`. I think it may be best to move the new code in front of the `if target in ['html', 'xhtml']` line since there is nothing target-specific in the new code anymore (which is nice).

Reply:
In TeX output we still to have to remove links as making them work there will be too hard.

Reply:
I agree. Still, the code for removing links should only exist once.

Reply:
And it will be (just in `tex` target branch) - I have this prepared already in other branch.